src/renderers/WebGLRenderer.js

```diff
@@ -1268,7 +1281,6 @@ function WebGLRenderer( parameters ) {
 	function renderObject( object, scene, camera, geometry, material, group ) {
 
 		object.onBeforeRender( _this, scene, camera, geometry, material, group );
-		currentRenderState = renderStates.get( scene, _currentArrayCamera || camera );
```

Comment:
This is not necessary anymore since the pervious render state is now always restored and the end of `render()`, right?

Reply:
Yep, that's right.